include/megaapi_impl.h

```diff
@@ -1102,6 +1108,8 @@ class MegaRequestPrivate : public MegaRequest
         MegaTextChatList *chatList;
 #endif
         MegaStringMap *stringMap;
+        MegaStringListMap *stringListMap;
+        MegaStringTable *stringTable;
```

Comment:
maybe `=nullptr;` now we have c++11, to make it easy to be sure these are appropriately set in the constructor?

Reply:
actually, as these are owning pointers and we now have c++11, we could use unique_ptr<> (which would make the constructor, setters, and destructor code simpler)

Reply:
Since there are many other pointers I want to keep it consistent and initialize in the constructor bodies and keep raw pointers. However, I did forget to delete in the destructor. Adding that...